src/formatFileContent.js

```diff
@@ -39,7 +39,21 @@ class LanguageServiceHost {
   getScriptSnapshot() {
     return ts.ScriptSnapshot.fromString(this.content);
   }
-}
+  readFile(fileName, encoding) {
+      if (fileName === this.fileName) {
+        return this.content
+      } else {
+        return ts.sys.readFile(fileName, encoding);
+      }
+  }
+  fileExists(fileName) {
+      if (fileName === this.fileName) {
+        return true;
+      } else {
+        return ts.sys.fileExists(fileName);
+      }
```

Comment:
nitpick; unnecessary else

```suggestion
      if (fileName === this.fileName) {
        return true;
      }
      
      return ts.sys.fileExists(fileName);
```